pom.xml

```diff
@@ -3,11 +3,11 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>com.trueffect</groupId>
-    <artifactId>MovieProyectWebV10.1</artifactId>
+    <artifactId>MovieProyectWebV15</artifactId>
```

Comment:
Can you explain the reason why this is changing? the `artifactId` should not change, it is the version the one that usually changes. Since you are not moving from one version to another, and you are not sharing your module as a dependency in other modules you can fix this to lets say `max-movie`